deepchecks/nlp/datasets/classification/tweet_emotion.py

```diff
@@ -28,13 +28,45 @@
 
 _FULL_DATA_URL = 'https://ndownloader.figshare.com/files/39265559'
 _EMBEDDINGS_URL = 'https://ndownloader.figshare.com/files/39264332'
+_PROPERTIES_URL = 'https://ndownloader.figshare.com/files/39460924'
 _PREDICTIONS_URL = 'https://ndownloader.figshare.com/files/39264461'
 
 ASSETS_DIR = pathlib.Path(__file__).absolute().parent.parent / 'assets' / 'tweet_emotion'
 _target = 'label'
 
 
-def load_data(data_format: str = 'TextData', as_train_test: bool = True) -> \
+def load_properties(as_train_test: bool = True) -> t.Union[pd.DataFrame, t.Tuple[pd.DataFrame, pd.DataFrame]]:
+    """Load and return the properties of the tweet_emotion dataset.
```

Comment:
Add license for dataset.

Reply:
I have no idea where its from... You approved Nadav's PR that merged it 😅

Reply:
missed it back then, let's cite the appropriate papers from https://huggingface.co/datasets/tweet_eval#licensing-information